### PR TITLE
adding my junit 5 assignment to the repository

### PR DESCRIPTION
**Pull request by Rahulkrsinhaa**
please review my code

### CODE REVIEW

File: src/test/java/juint_5/PersonTest.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class PersonTest {
+    Person person;
+
+    @BeforeEach
+    public void createInstance() {
```
Comment (anup0060) [inline src/test/java/juint_5/PersonTest.java]: method name should be setup

File: src/test/java/juint_5/PersonTest.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        assertEquals("", person.getName());
+
+        //testing getname with white spaces
+        person.setName("         ");
+        assertEquals("", person.getName());
+    }
+
+
+    @Test
+    public void testGetAge() throws InvalidAgeException {
```
Comment (anup0060) [inline src/test/java/juint_5/PersonTest.java]: split in separate test methods , don't club multiple scenarios.

File: src/test/java/juint_5/PersonTest.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    public void testNegativeAge() {
+        InvalidAgeException e = assertThrows(InvalidAgeException.class, () -> {
+            person.setAge(-10);
+        });
+        assertEquals("age cannot be null or in negative", e.getMessage());
+    }
+
+
+    @Test
+    public void testSetAgeZero() throws InvalidAgeException {
```
Comment (anup0060) [inline src/test/java/juint_5/PersonTest.java]: test method names should be correct , see junit 4 review comments

**Review by anup0060 — CHANGES_REQUESTED** [review]

**Comment by Rahulkrsinhaa** [comment]
I have worked on the above reviews please go through it and let me know if I am still lacking in something

**Review by anup0060 — APPROVED** [review]

**Pull request merged**